test/routable_component_routes.js

```diff
@@ -63,31 +63,31 @@ describe('RoutableComponentRoutes', function() {
   });
   /** @test {RoutableComponentRoutes#constructor} */
   context('constructed with', function() {
-    subject(function() { return new RoutableComponentRoutes(this.routing_classes) });
+    subject(function() { return new RoutableComponentRoutes(this.routingClasses) });
     context('single routing classes', function() {
-      lazy('routing_classes', function() { return MyRouting });
+      lazy('routingClasses', function() { return MyRouting });
       it('surely initialized', function() { assert(this.subject instanceof RoutableComponentRoutes) });
     });
     context('multiple routing classes', function() {
-      lazy('routing_classes', function() { return [MyRouting, MyRoutingWithFrom] });
+      lazy('routingClasses', function() { return [MyRouting, MyRoutingWithFrom] });
       it('surely initialized', function() { assert(this.subject instanceof RoutableComponentRoutes) });
     });
   });
-  /** @test {RoutableComponentRoutes#include_route} */
-  context('include_route with', function() {
+  /** @test {RoutableComponentRoutes#includeRoute} */
+  context('includeRoute with', function() {
     subject(function() {
       const routes = new RoutableComponentRoutes();
-      routes.include_route(this.routing_classes);
+      routes.includeRoute(this.routingClasses);
```

Comment:
__[eslint]__
* Error - Unexpected 'this'. (no-invalid-this)

<!-- Comment by [SideCI](https://sideci.com) -->